recipes/parserutils/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "parserutils" %}
+{% set version = "2.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/parserutils-{{ version }}.tar.gz
+  sha256: f927e69779d81db508db98e6e8ec331f90ff31ff2868161d1cce30fffe92bec3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - mock
```

Comment:
This should either be moved under `run`, or alternatively just remove the `parserutils.tests` from under the test imports. I don't think it's important that the tests be importable for users of the package, but I could be wrong?

Reply:
@dopplershift Back from a tiny vacation. Changes pushed and everything looks clean in the CI. Much appreciate the support on this.

Reply:
The current CI failures at due to a CI related failure and nothing to do with the recipe. I am not seeing the option to re-run CI. Perhaps I should use a rerender?